changelog.md

```diff
@@ -123,6 +123,8 @@ echo f
   retrieving the verified certificate chain of the peer we are connected to
   through an SSL-wrapped `Socket`/`AsyncSocket`.
 - Added `distinctBase` overload for values: `assert 12.MyInt.distinctBase == 12`
+- Added `jsconsole.trace`, `jsconsole.table`, `jsconsole.exception` for JavaScript target.
```

Comment:
typos: jconsole=>jsconsole

Reply:
Typo how, where?, the module is named `jsconsole.nim`, changelog says `jsconsole`.

Reply:
I hallucinated, super weird, sry.